backend/benefit/applications/migrations/0017_searchable_encrypted_social_security_number.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.4 on 2021-09-03 11:07
+
+from django.db import migrations
+import encrypted_fields.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("applications", "0016_benefit_amount"),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
According to `django-searchable-encrypted-field` document: https://gitlab.com/guywillett/django-searchable-encrypted-fields#migrations-add-searchencryptedfields-to-your-model-dont-alter-existing-fields. It is recommended to create a new encrypted field and migrate the old data to the new one.

Although we don't have any actual data anywhere, I think it's still better to at least create a whole new EncryptedField than renaming the old one

Reply:
OK good catch, I added a pair of migrations to delete/add the field. (removing and then adding the same field in the same migration didn't seem to work)